server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/duplication/IntegrateCrossProjectDuplications.java

```diff
@@ -60,6 +61,8 @@
   public IntegrateCrossProjectDuplications(Configuration config, DuplicationRepository duplicationRepository) {
     this.config = config;
     this.duplicationRepository = duplicationRepository;
+    config.getBoolean(CoreProperties.CPD_CROSS_PROJECT).filter(s -> s).ifPresent(s ->
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "CPD_CROSS_PROJECT"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)